### PR TITLE
Update documentation with Ingress Versioning capabilities.

### PR DESCRIPTION
**Pull request by gargravarr**
This Pull Request updates the current README and HOWTOs for Ingress Versioning, which now is what the StackSet controller uses when generating Stack resources.

I also improved some wording and took the chance to update the diagram to uses Mermaid syntax.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,23 +38,11 @@ application. The `StackSet` also allows defining a "global" load balancer
...
-                                 +--+--------+--------+--+
-                                    | 0%     | 20%    | 80%
-                      +-------------+        |        +------------+
-                      |                      |                     |
-            +---------v---------+  +---------v---------+  +--------v----------+
-            |                   |  |                   |  |                   |
-            |       Stack       |  |       Stack       |  |      Stack        |
-            |     Version 1     |  |     Version 2     |  |     Version 3     |
-            |                   |  |                   |  |                   |
-            +-------------------+  +-------------------+  +-------------------+
```
Comment (mikkeloscar) [inline README.md:57]: My beautiful ascii art 🥹

/jk

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
Thanks @gargravarr !

**Pull request merged**